element_array_ephys/ephys_acute.py

```diff
@@ -23,30 +23,14 @@
 
 def activate(ephys_schema_name, probe_schema_name=None, *, create_schema=True,
              create_tables=True, linking_module=None):
-    """
-    activate(ephys_schema_name, probe_schema_name=None, *, create_schema=True, create_tables=True, linking_module=None)
-        :param ephys_schema_name: schema name on the database server to activate the `ephys` element
-        :param probe_schema_name: schema name on the database server to activate the `probe` element
-         - may be omitted if the `probe` element is already activated
-        :param create_schema: when True (default), create schema in the database if it does not yet exist.
-        :param create_tables: when True (default), create tables in the database if they do not yet exist.
-        :param linking_module: a module name or a module containing the
-         required dependencies to activate the `ephys` element:
-            Upstream tables:
-                + Session: table referenced by EphysRecording, typically identifying a recording session
-                + SkullReference: Reference table for InsertionLocation, specifying the skull reference
-                 used for probe insertion location (e.g. Bregma, Lambda)
-            Functions:
-                + get_ephys_root_data_dir() -> list
-                    Retrieve the root data directory - e.g. containing the raw ephys recording files for all subject/sessions.
-                    :return: a string for full path to the root data directory
-                + get_session_directory(session_key: dict) -> str
-                    Retrieve the session directory containing the recorded Neuropixels data for a given Session
-                    :param session_key: a dictionary of one Session `key`
-                    :return: a string for full path to the session directory
-                + get_processed_root_data_dir() -> str:
-                    Retrieves the root directory for all processed data to be found from or written to
-                    :return: a string for full path to the root directory for processed data
+    """Activates the `ephys` and `probe` schemas. 
+
+    Args:
+        ephys_schema_name (str): A string containing the name of the ephys schema.
+        probe_schema_name (str): A string containing the name of the probe scehma.
+        create_schema (bool): If True, schema will be created in the database.
+        create_tables (bool): If True, tables related to the schema will be created in the database.
+        linking_module (str): A string containing the module name or module containing the required dependencies to activate the schema.
```

Comment:
Please readd upstream items following [this convention](https://github.com/datajoint/element-deeplabcut/blob/main/element_deeplabcut/model.py#L44-L53)

Reply:
Done